### PR TITLE
Preparation for release v2023.1.4 (CLI)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Preparation for release.

## Changes

Updates files `Changes`, `Makefile.PL` and `CLI.pm`.

## How to test this PR

Review the changes.

### CODE REVIEW

**Review by mattias-p — APPROVED** [review]

**Review by marc-vanderwal — APPROVED** [review]

File: Changes [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Release history for Zonemaster component Zonemaster-CLI 
 
 
+v6.0.3 2023-09-05 (public fix version)
+
+ [Fixes]
+ - This version contains no real changes. It has been created to
+   require a higher (fixed) version of Zonemaster-Engine.
```
Comment (pnax) [inline Changes:8]: Please include https://github.com/zonemaster/zonemaster-cli/pull/351.

**Review by pnax — CHANGES_REQUESTED** [review]

File: Changes [hunk 2]
```diff
@@ -1,6 +1,13 @@
 Release history for Zonemaster component Zonemaster-CLI 
 
 
+v6.0.3 2023-09-05 (public fix version)
+
+ [Fixes]
+ - Fixes a spelling error (text) in the zonemaster-cli scrip (#351).
```
Comment (mattias-p) [inline Changes:7]:
```suggestion
 - Fixes a spelling error (text) in the zonemaster-cli script (#351).
```

**Review by mattias-p — APPROVED** [review]

**Review by marc-vanderwal — APPROVED** [review]

**Comment by matsduf** [comment]
@pnax, please re-review.

File: Changes [hunk 3]
```diff
@@ -1,6 +1,13 @@
 Release history for Zonemaster component Zonemaster-CLI 
 
 
+v6.0.3 2023-09-05 (public fix version)
```
Comment (pnax) [inline Changes:4]: Shouldn't the date be updated?

**Review by pnax — COMMENTED** [review]
beside the date, lgtm

**Review by marc-vanderwal — APPROVED** [review]

File: Changes (hunk 3 above)
Comment (marc-vanderwal) [reply Changes]: The date has been changed.

**Pull request merged**